vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -52,13 +52,15 @@ IRONIC_IMAGE_DIR: "{{ lookup('env', 'IRONIC_IMAGE_DIR') | default('/opt/metal3-d
 IRONIC_ENDPOINT_BRIDGE: "{{ lookup('env', 'CLUSTER_PROVISIONING_INTERFACE') | default('provisioning')}}"
 
 # Distibution specific environment variables
+SOURCE_IMAGE_NAME: "{{ lookup('env', 'IMAGE_NAME') | default('bionic-server-cloudimg-amd64.img', true)}}"
 IMAGE_NAME: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('bionic-server-cloudimg-amd64-raw.img', true)}}"
 IMAGE_LOCATION: 'https://cloud-images.ubuntu.com/bionic/current'
 IMAGE_URL: "http://172.22.0.1/images/{{ IMAGE_NAME }}"
 IMAGE_CHECKSUM: "http://172.22.0.1/images/{{ IMAGE_NAME }}.md5sum"
 
-IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64-raw.qcow2', true)}}"
-IMAGE_LOCATION_CENTOS: 'https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2'
+SOURCE_IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2', true)}}"
+IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64-raw.img', true)}}"
+IMAGE_LOCATION_CENTOS: 'https://cloud.centos.org/centos/8/x86_64/images'
```

Comment:
Same thing here, do we need `IMAGE_NAME_CENTOS `separately ? If no, can we instead use the old vars but replace lookup part?

Reply:
`SOURCE_IMAGE_NAME_CENTOS` is the name of the image we want to download. `IMAGE_NAME_CENTOS` holds the name of the raw image. I think we need both?

Reply:
yes we need both. But the variables could be renamed to be more explicit if needed

Reply:
My apologies, I misunderstood it.
Then, if we are downloading qcow2 image and converting it to raw image on the fly, what is the purpose of having lookup on `IMAGE_NAME_CENTOS ` var? Is this var declared just to initialize raw image name? If yes, maybe we shouldn't keep lookup part?

Also, can we rename ` IMAGE_NAME_CENTOS` vars to for example `RAW_IMAGE_NAME_CENTOS` and keep the `IMAGE_NAME_CENTOS` for qcow2 image to be more explicit ?

Reply:
agreed with the last point, we need the lookup on all variables as they might be defined by the user at the metal3-dev-env level, so we should not assume anything in the ansible playbook. That's because the image (at least in CI) is already downloaded beforehand, so we need to know the name of the file